orchestrator-salt/src/main/resources/salt/salt/sssd/ipa.sls

```diff
@@ -37,4 +39,25 @@ restart-sssd-if-reconfigured:
     - mode: 700
 
 include:
-    - sssd.ssh
\ No newline at end of file
+    - sssd.ssh
+
+{%- if "manager_server" in grains.get('roles', []) %}
+
+create_cm_keytab_generation_script:
+  file.managed:
+    - name: /opt/salt/scripts/generate_cm_keytab.sh
+    - source: salt://sssd/template/generate_cm_keytab.j2
+    - makedirs: True
+    - template: jinja
+    - context:
```

Comment:
please switch to 'env' solution like here: https://github.com/hortonworks/cloudbreak/blob/master/orchestrator-salt/src/main/resources/salt/salt/sssd/ipa.sls#L16
so we don't have secrets in the generated script